acceptance/localcluster/docker.go

```diff
@@ -0,0 +1,295 @@
+// Copyright 2015 The Cockroach Authors.
```

Comment:
I'm surprised this functionality isn't available through some existing package. 

Assuming it's not, seems like we should make this a separate package so that other people can use it. I imagine this functionality would be in at least slight, but growing, demand.